src/shadowbox/infrastructure/filesystem_text_file.ts

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import * as fs from 'fs';
-import {TextFile} from '../model/text_file';
+import {TextFile} from '../infrastructure/text_file';
```

Comment:
It's in the same directory:
```suggestion
import {TextFile} from './text_file';
```